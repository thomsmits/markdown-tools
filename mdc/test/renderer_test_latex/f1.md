# Chapter
## Slide

> Test _"Quoted"_ or _unqoted_

Test _"Quoted"_ or _unqoted_

---

> Test _"Quoted"_ or _unqoted_

Test _"Quoted"_ or _unqoted_
