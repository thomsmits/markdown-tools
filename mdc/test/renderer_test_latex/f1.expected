# Chapter

## Slide

\quotedns{Test \strong{\enquote{Quoted}} or \strong{unqoted}
}

Test \strong{\enquote{Quoted}} or \strong{unqoted}\vspace{0.1mm}

\quotedns{Test \strong{\enquote{Quoted}} or \strong{unqoted}
}

Test \strong{\enquote{Quoted}} or \strong{unqoted}\vspace{0.1mm}
